health-monitors/system-services-monitor/system_services_monitor/tests/test_cli.py

```
@@ -0,0 +1,138 @@
+
+        assert result.exit_code == 0
+        assert processor.call_args.kwargs["socket_path"] == "/run/from-env.sock"
+        wk = watcher.call_args.kwargs
+        assert wk["poll_interval"] == 15  # CHECK_INTERVAL
+        assert wk["boot_grace_period"] == 120  # BOOT_GRACE_PERIOD
+        assert wk["flap_window"] == 300  # FLAP_WINDOW
+        assert wk["flap_threshold"] == 5  # FLAP_THRESHOLD
+
+    def test_cli_arg_overrides_env(self, runner: CliRunner) -> None:
+        """An explicit CLI flag wins over the corresponding env var."""
+        with _mock_runtime() as (watcher, processor):
```

Ruff (0.15.21)

[warning] Unpacked variable `watcher` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)